src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -301,8 +346,7 @@ public static string GetModifiedJsonString(string[] reps, string enumString)
   },
   ""data-source"": {
     ""database-type"": " + enumString + @",
-    ""connection-string"": ""server=dataapibuilder;database=" + reps[++index % reps.Length] + @";uid=" + reps[++index % reps.Length] + @";Password=" + reps[++index % reps.Length] + @";"",
-    ""resolver-config-file"": """ + reps[++index % reps.Length] + @"""
```

Comment:
was `""resolver-config-file"": """ + reps[++index % reps.Length] + @"""` legacy code? I suppose since `resolver-config-file` is no longer used.

Reply:
yes, correct.